contracts/src/libs/ComputableComponents.sol

```diff
@@ -54,4 +49,20 @@ library ComputableComponents {
     function kind(bytes32 logicRef, bytes32 labelRef) internal pure returns (bytes32 k) {
         k = sha256(abi.encode(logicRef, labelRef));
     }
+
+    /// @notice Computes the randomness for the commitment
+    /// @param resource The resource whose randomness we compute
+    /// @return randCm The randomness for the resource commitment
+    function rcm(Resource memory resource) internal pure returns (bytes32 randCm) {
+        bytes17 prfExpandPersonalization = 0x52495343305f457870616e645365656401;
```

Comment:
This constant should be made an actual `constant` in this library.
We should also explain where this is coming from / how it is derived.